src/test/regress/sql/failure_setup.sql

```diff
@@ -1,5 +1,5 @@
 SELECT citus.mitmproxy('conn.allow()');
 
 -- add the workers
```

Comment:
Do we support failure tests for hyperscale? If not, maybe revert the changes in `failure_*` files to fix them properly when we want to support failure tests.